sklearnex/tests/test_memory_usage.py

```diff
@@ -124,6 +132,32 @@ def gen_functions(functions):
 ORDER_DICT = {"F": np.asfortranarray, "C": np.ascontiguousarray}
 
 
+DUMMY_ESTIMATOR = {}
+if _is_dpc_backend:
+
+    from onedal.datatypes._data_conversion import from_table, to_table
+
+    class DummyEstimatorWithTableConversions(BaseEstimator):
+
+        def fit(self, X, y=None):
+            sua_iface, _, _ = _get_sycl_namespace(X)
+            X_table = to_table(X, sua_iface=sua_iface)
```

Comment:
Should we store ```X_table``` and ```y_table``` as ```self.X_table_``` and ```self.y_table_``` otherwise sklearn's ```check_is_fitted``` will show it isn't fitted, and the cleanup of the memory will occur at ```DummyEstimatorWithTableConversions``` deallocation/deletion rather than immediately after calling fit?

Reply:
Thank you! Very good comment.
I have updated the dummy estimator and added comment.